operator/v1/types_console.go

```diff
@@ -55,15 +55,22 @@ type ConsoleSpec struct {
 	// +optional
 	Route ConsoleConfigRoute `json:"route"`
 	// plugins defines a list of enabled console plugin names.
+	// +listType=atomic
+	// +kubebuilder:validation:MaxItems=64
+	// +kubebuilder:validation:XValidation:rule="self.all(x, self.exists_one(y, x == y))",message="each plugin name must be unique"
 	// +optional
-	Plugins []string `json:"plugins,omitempty"`
+	Plugins []PluginName `json:"plugins,omitempty"`
```

Comment:
When did this API ship?

Is there a finite, enumerable list of plugins, that we as RH control?

What happens if the list is not already completely unique? Have you tested this?